Example/ViewController.swift

```diff
@@ -578,7 +557,7 @@ extension ViewController: VisualInstructionDelegate {
 // MARK: Free driving
 extension ViewController {
     func trackLocations(mapView: NavigationMapView) {
-        let dataSource = PassiveLocationDataSource(directions: Settings.directions)
+        let dataSource = PassiveLocationDataSource(directions: Directions.shared)
```

Comment:
Nit: This is the default value already:

```suggestion
        let dataSource = PassiveLocationDataSource()
```